app/src/androidTest/java/com/github/polypoly/app/network/RemoteDBTest.kt

```diff
@@ -1,49 +1,92 @@
 package com.github.polypoly.app.network
 
-import com.github.polypoly.app.commons.PolyPolyTest
-import com.github.polypoly.app.game.GameLobby
-import com.github.polypoly.app.game.user.User
+import com.github.polypoly.app.base.user.Skin
+import com.github.polypoly.app.base.user.Stats
+import com.github.polypoly.app.base.user.User
 import com.github.polypoly.app.global.GlobalInstances.Companion.remoteDB
+import com.google.firebase.database.DatabaseReference
+import com.google.firebase.database.ktx.database
+import com.google.firebase.ktx.Firebase
 import org.junit.Assert.*
+import org.junit.Before
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.junit.runners.JUnit4
 import java.util.*
+import java.util.concurrent.CompletableFuture
 import java.util.concurrent.ExecutionException
 import java.util.concurrent.TimeUnit
-import kotlin.reflect.KClass
 
 @RunWith(JUnit4::class)
-class RemoteDBTest: PolyPolyTest(true, false) {
-
-    private fun <T : Any> classCanBeStoredInDB(clazz: KClass<T>) {
-        val noArgsKey = "no-args-obj"
-        val noArgsConstructor = clazz.java.getDeclaredConstructor()
-        val noArgsInstance = noArgsConstructor.newInstance()
-        addDataToDB(listOf(noArgsInstance), listOf(noArgsKey))
-        assertEquals(
-            noArgsInstance,
-            remoteDB.getValue(noArgsKey, clazz).get(TIMEOUT_DURATION, TimeUnit.SECONDS)
-        )
```

Comment:
I think something went wrong with the merge, this should be kept and the code below (marked added) is outdated.

Reply:
Okay it actually looks like you rolled back the file back to its previous version during merging so something definitely went wrong.

Reply:
ok I copy your `RemoteDBTest` from the main